Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "delaunator"
```

Comment:
Since this is contract breaking, I wonder if we should bump this to 1.0.0 (major version increase for breaking changes), or at least 0.3.0 if we don't want to change the major version.

Reply:
Let's do `v0.3.0` — I think you can expect some breaking changes in minor versions if it's in v0.x. I'd target v1.0 for the fully robust version.

Reply:
Makes sense. I'll update to v0.3.0